intermediate-exercises/src/components/03-FetchGithubData.js

```diff
@@ -23,41 +23,66 @@ import React, { Component } from 'react';
 const GithubRepos = ({ repos }) => {
   return (
     <ul>
-      {/* Task: The list of repos here */}
+      {repos.map(({ id, name }) => (
+        <li key={id}>{name}</li>
+      ))}
     </ul>
   );
-}
+};
 
 // Task: Open the console in the browser. There will be a warning
 // about incorrect prop type for user.
 // Define the correct prop type for the prop `repos`
 GithubRepos.propTypes = {
-
+  repos: PropTypes.array.isRequired
 };
 
 /* eslint-disable react/no-multi-comp */
 class UsernameForm extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      username: '',
-      repos: [],
+      username: "",
+      repos: []
     };
   }
+  getUserRepos = async () => {
```

Comment:
Can this be made as normal function with bind in the constructor.

Reply:
It is fine I believe that's simply another way.